versions.props

```diff
@@ -19,7 +19,6 @@ commons-io:commons-io = 2.1
 io.atomix:* = 1.0.0-rc9
 io.atomix.catalyst:* = 1.1.2
 io.atomix.copycat:* = 1.2.0
-io.dropwizard.metrics:metrics-* = 3.1.1
```

Comment:
Is the assumption that folks using newer versions will explicitly exclude `com.codahale.metrics` and pull in their own `io.dropwizard.metrics`?